user/languages/python.md

```diff
@@ -19,12 +19,20 @@ Travis CI supports Python versions 2.6, 2.7, 3.2, 3.3, 3.4, 3.5, as well as rece
 ```yaml
 language: python
 python:
-  - "2.6"
-  - "2.7"
-  - "3.2"
-  - "3.3"
-  - "3.4"
-  - "3.5"
+  - "2.6"     # Python 2.6.9
+  - "2.7"     # Python 2.7.9 for sudo: false
+              # Python 2.7.12 for sudo: required
+  - "2.7.12"  # latest 2.7 available
+  - "3.2"     # Python 3.2.5 for sudo: false
+              # Python 3.2.6 for sudo: required
+  - "3.2.6"   # latest 3.2 available
+  - "3.3"     # Python 3.3.5 for sudo: false
+              # Python 3.3.6 for sudo: required
+  - "3.3.6"   # latest 3.3 available
+  - "3.4"     # Python 3.4.2 for sudo: false
+              # Python 3.4.4 for sudo: required
+  - "3.4.5"   # latest 3.4 available
+  - "3.5"     # Python 3.5.2
   - "3.5-dev" # 3.5 development branch
   - "3.6-dev" # 3.6 development branch
```

Comment:
According to @JordonPhillips in https://github.com/travis-ci/travis-ci/issues/7085 3.6 is actually the release version - [3.6.0](https://www.python.org/downloads/release/python-360/).

Documentation should be updated to reflect that.